src/renderer/views/About/About.js

```diff
@@ -21,7 +21,7 @@ export default defineComponent({
         {
           icon: ['fab', 'github'],
           title: this.$t('About.Source code'),
-          content: `<a href="https://github.com/FreeTubeApp/FreeTube">GitHub: FreeTubeApp/FreeTube</a><br>${this.$t('About.Licensed under the')} <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">${this.$t('About.AGPLv3')}</a>`
+          content: `<a href="https://github.com/FreeTubeApp/FreeTube" lang="en">GitHub: FreeTubeApp/FreeTube</a><br>${this.$t('About.Licensed under the')} <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">${this.$t('About.AGPLv3')}</a>`
```

Comment:
Should `lang="en"` be added to other links too?

Reply:
Any link in particular that you have in mind? I added it to this one because the display text is English, form what I can tell the rest of them on the about page either have the link itself as the display text or are translated, but I might have missed something.

Reply:
Oh they are translated? Did not check :)